### PR TITLE
{math}[GCCcore/11.3.0] matlab-proxy v0.5.4 and aiohttp v3.8.3 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 2 out of 4 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/ec09def3de4da7a95bc1327ea2ce987e for a full test report.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso
EB_ARGS='nodejs-16.15.1-GCCcore-11.3.0.eb matlab-proxy-0.5.4-GCCcore-11.3.0.eb -r'

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/7f22b3170f8b76a4f5350f80819066b5 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
eb-u2204 - Linux Ubuntu 22.04, x86_64, AMD Ryzen 9 3900X 12-Core Processor, Python 3.10.6
See https://gist.github.com/7a55095f1bf4edbfa0b1bcd309fad257 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
eb-rocky9.zao.se - Linux Rocky Linux 9.0 (Blue Onyx), x86_64, AMD Ryzen 9 3900X 12-Core Processor, Python 3.9.10
See https://gist.github.com/4dd3b9c4b81c07e49ce8b8e3cceec795 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.5, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/eb7aab8f260b5e966912dc8f8a9090dd for a full test report.

**Review by Micket — CHANGES_REQUESTED** [review]
I think we want to include the patch to pick up on the license server directly, like i did with the patch for jupyter-matlab-proxy

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by Micket** [comment]
I still want the patch :)

**Comment by akesandgren** [comment]
Which license server? there is no lic for the proxy stuff.

**Comment by Micket** [comment]
> Which license server? there is no lic for the proxy stuff.

Have you tried the proxy? It will force each user to manually enter the license server for MATLAB (even when you could just launch matlab directly and it would find it's own license server. 

The patch enter the license information directly from EB.

**Comment by akesandgren** [comment]
I don't know how to use it so no, I haven't tried it

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/e114ca960936536ab08ebfcf2c1facac for a full test report.

File: easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+easyblock = "PythonBundle"
+
+name = 'matlab-proxy'
+version = '0.5.4'
+
+homepage = 'https://github.com/mathworks/matlab-proxy'
+description = 'MATLAB Integration for Jupyter'
```
Comment (lexming) [inline easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb:7]: `matlab-proxy` is independent from Jupyter, please update the description.

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb]: I blame @Micket who made the original one :-) I didn't even check the description :-)

Fixed

Comment (lexming) [reply easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb]: I agree, in case of doubt always blame @Micket :laughing:

Comment (Micket) [reply easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb]: The *jupyter*-matlab-proxy is actually a jupyter thing! Though i'm actually unsure in what way. There doesn't seem to be any difference at all between these two.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/m/matlab-proxy/matlab-proxy-0.5.4-GCCcore-11.3.0.eb]: My understanding is that `matlab-proxy` does not provide the integration with jupyter. It is just the http proxy to the web server of MATLAB. `jupyter-matlab-proxy` is matlab-proxy + jupyter-server-proxy to have that running inside JupyterLab.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c2 - Linux Rocky Linux 8.5, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/00548e8313085fa30c00a5ab975eb0e5 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by Micket** [comment]
I didn't have any problems loading MATLAB after matlab-proxy either (as the lmod message would indicate), but perhaps it's an issue on some operating systems? It's fine this way as well, doesn't hurt to load it first.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @akesandgren!

**Comment by akesandgren** [comment]
Loading MATLAB first is just to make sure EBROOTMATLAB is set once it tries to pick up the license file.

**Pull request merged**